drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -532,7 +536,7 @@ def introspect_custom_data_element_definitions(set_definition_identifier: false)
       cded.label = ActionView::Base.full_sanitizer.sanitize(item.readonly_text || item.brief_text || item.text || key.humanize)
 
       # If specified, set the definition identifier to specify that this CustomDataElementDefinition is ONLY collected by this form type.
-      cded.definition_identifier = identifier if set_definition_identifier
```

Comment:
@gigxz , Theron and I realized we were confused about this - was it here proactively as a placeholder, or is `definition_identifier` actually something different?

Reply:
no I was just trying to be helpful! my bad